packages/editor-ui/src/components/NodeWebhooks.vue

```diff
@@ -195,12 +193,27 @@ export default defineComponent({
 			return '';
 		},
 		isWebhookMethodVisible(webhook: IWebhookDescription): boolean {
+			try {
+				const method = this.workflowHelpers.getWebhookExpressionValue(webhook, 'httpMethod', false);
+				if (Array.isArray(method) && method.length !== 1) {
+					return false;
+				}
+			} catch (error) {}
+
 			if (typeof webhook.ndvHideMethod === 'string') {
 				return !this.workflowHelpers.getWebhookExpressionValue(webhook, 'ndvHideMethod');
 			}
 
 			return !webhook.ndvHideMethod;
 		},
+
+		getWebhookHttpMethod(webhook: IWebhookDescription): string {
+			const method = this.workflowHelpers.getWebhookExpressionValue(webhook, 'httpMethod', false);
```

Comment:
Should there be a catch here? asking because you are catching the same call on line 201

You also might be able to extract some common logic between `isWebhookMethodVisible` and `getWebhookHttpMethod`

Reply:
yes, as `method` could be string or array